.travis.yml

```diff
@@ -10,7 +10,7 @@ notifications:
 
 env:
   global:
-    - SKEEMA_TEST_IMAGES="mysql:5.6,mysql:5.7"
+    - SKEEMA_TEST_IMAGES="mysql:5.5,mysql:5.6"
```

Comment:
Ah good call, that is definitely easier than getting the whole test suite setup locally 👍 

After merging this PR, I'll restore this line to 5.6+5.7 in branch fk-order-mysql55 before merging that branch into master.